client/src/AddFriend.js

```diff
@@ -39,29 +40,31 @@ class AddFriend extends React.Component {
       })
   }
 
+  // TODO: prevent page reload on empty input
   handleAdd (e) {
     // check for empty inputs
     this.setState({ isLoading: true })
     if (!this.state.friendFName || !this.state.friendLName || !this.state.friendEmail) {
-      return this.setState({ status: 406 })
+      return this.setState({ alertType: 'emptyField' })
     }
     axios.post('/api/friendships', this.state)
       .then(res => {
         console.log('got to axios.post')
         if (res.status >= 200 && res.status < 300) {
           this.setState(state => {
             const friendships = state.friendships.concat(state.friendFName + ' ' + state.friendLName)
-            const status = 201
-            return { friendships, status }
+            const alertType = 'created'
+            return { friendships, alertType }
           })
         }
-        this.setState({ isLoading: false })
+
+        // This seems to be clearning the friend name before alert can pass the data into alert component.
+        this.setState({ isLoading: false, friendFName: '', friendLName: '', friendEmail: '' })
       })
       .catch(err => {
         if (err.response.status === 409) {
-          this.setState({ status: 409 })
+          this.setState({ alertType: 'friendshipExists', isLoading: false, friendFName: '', friendLName: '', friendEmail: '' })
```

Comment:
[nit] We can minimize code duplication by creating a short helper function

`resetForm = () => this.setState({ friendFName: '', friendLName: '', friendEmail: '' })`